### PR TITLE
work so far on the new user guide to kifi

### PR DESCRIPTION
**Pull request by 2is10**

### CODE REVIEW

**Comment by andrewconner** [comment]
Wow!

File: packrat/scripts/guide/spotlight.js [hunk 1]
```diff
@@ -0,0 +1,210 @@
+// @require styles/guide/spotlight.css
+// @require scripts/lib/underscore.js
+
+var Spotlight = Spotlight || (function (window, document) {
```
Comment (andrewconner) [inline packrat/scripts/guide/spotlight.js:4]: Love seeing this more modular code.

File: packrat/scripts/guide/svg_arrow.js [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+                 :                \ /  :     | of elliptical arc
+                 : . . . . . . . . V . :    _|    __
+                                   H                |
+                                                    | tip space height
+                       ,-----------------------,   -'
+                       | elHead                |
+                       |           •C          |
+                       |                       |
+                       '-----------------------'
+    */
```
Comment (andrewconner) [inline packrat/scripts/guide/svg_arrow.js:31]: Nice!

File: packrat/scripts/guide/svg_arrow.js [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+
+    // solving for parameter t and then radii a and b using these formulae from
+    // en.wikipedia.org/wiki/Ellipse#Parametric_form_in_canonical_position
+    // G.x === a * cos(t)
+    // G.y === b * (1 - sin(t))  // "1 -" transforms to standard Cartesian coordinates from SVG
+    // -tan(t) === b / (a * tan(phi))
+
+    var t = Math.asin(-G.y / (G.y + G.x * Math.tan(phi)));
+    var a = G.x / Math.cos(t);
+    var b = G.y / (1 - Math.sin(t));
```
Comment (2is10) [inline packrat/scripts/guide/svg_arrow.js:132]: @yingjieMiao @leogrim @martinraison Here’s where you helped. Thanks!

File: packrat/scripts/guide/svg_arrow.js [hunk 4]
```diff
@@ -0,0 +1,267 @@
...
+    return ['M', H.x, H.y, 'l', -HEAD_LENGTH, -HEAD_WIDTH / 2, 'l', 0, HEAD_WIDTH, 'z'].join(' ');
+  }
+
+  function ellipseArc(H, phi, T) {  // phi is the tangential angle (radians from positive x-axis)
+    var G = {  // point of tangency is the center of the arrow head base
+      x: H.x + HEAD_LENGTH * Math.cos(phi),
+      y: H.y - HEAD_LENGTH * Math.sin(phi)
+    };
+
+    // solving for parameter t and then radii a and b using these formulae from
```
Comment (andrewconner) [inline packrat/scripts/guide/svg_arrow.js:124]: correction: formulæ

File: packrat/scripts/guide/svg_arrow.js (hunk 3 above)
Comment (yingjieMiao) [inline packrat/scripts/guide/svg_arrow.js:132]: welcome!

**Pull request merged**